test/src/test/java/org/corfudb/integration/LogReplicationAbstractIT.java

```diff
@@ -387,20 +389,28 @@ public void setupActiveAndStandbyCorfu() throws Exception {
         }
     }
 
-    public void openMaps(int mapCount) throws Exception {
+    public void openMaps(int mapCount, boolean diskBased) throws Exception {
         mapNameToMapActive = new HashMap<>();
         mapNameToMapStandby = new HashMap<>();
 
         for(int i=1; i <= mapCount; i++) {
             String mapName = TABLE_PREFIX + i;
 
+            Path pathActive = null;
+            Path pathStandby = null;
+
+            if (diskBased) {
+                pathActive = Paths.get(com.google.common.io.Files.createTempDir().getAbsolutePath());
+                pathStandby = Paths.get(com.google.common.io.Files.createTempDir().getAbsolutePath());
+            }
+
             Table<Sample.StringKey, Sample.IntValueTag, Sample.Metadata> mapActive = corfuStoreActive.openTable(
                     NAMESPACE, mapName, Sample.StringKey.class, Sample.IntValueTag.class, Sample.Metadata.class,
-                    TableOptions.fromProtoSchema(Sample.IntValueTag.class));
+                    TableOptions.fromProtoSchema(Sample.IntValueTag.class, TableOptions.builder().persistentDataPath(pathActive).build()));
```

Comment:
If pathActive is null, will this pass a null as the second variable or an empty tableOption?

Reply:
I thought it will pass an TableOption object with `null` value set for `persistentDataPath` field. Since we are using an Optional getter for this field, it will make the check in `TableOptions` fail:
```java
public Optional<Path> getPersistentDataPath() {
    return Optional.ofNullable(persistentDataPath);
}
```
```java
if (tableOptions != null && tableOptions.getPersistentDataPath().isPresent()) {
    tableOptionsBuilder.persistentDataPath((Path) tableOptions.getPersistentDataPath().get());
}
```